lsp-julia.el

```diff
@@ -25,33 +30,35 @@
 
 (defcustom lsp-julia-timeout 30
   "Time before lsp-mode should assume julia just ain't gonna start."
+  :type 'number
   :group 'lsp-julia)
 
 (defun lsp-julia--get-root ()
-  "Try to find the package directory by searching for a .gitignore file.
-If no .gitignore file can be found use the default directory "
-  (let ((dir (locate-dominating-file default-directory ".gitignore")))
-    (if dir
-        (expand-file-name dir)
-      default-directory)))
+  (let ((dir (locate-dominating-file default-directory "Project.toml")))
+    (if dir (expand-file-name dir)
+      (expand-file-name lsp-julia-default-environment))))
 
 (defun lsp-julia--rls-command ()
-  `(,lsp-julia-command ,@lsp-julia-flags "-e using LanguageServer; server = LanguageServer.LanguageServerInstance(STDIN, STDOUT, false); server.runlinter = true; run(server);"))
-
+  `(,lsp-julia-command
+    ,@lsp-julia-flags
+    ,(concat "-e using LanguageServer, Sockets, SymbolServer;"
+             " server = LanguageServer.LanguageServerInstance("
+             " stdin, stdout, false,"
+             " \"" (lsp-julia--get-root) "\","
```

Comment:
I'm a bit fuzzy on string literals in elisp. This inserts the root package directory as an argument to `LanguageServerInstance`?

Reply:
exactly